app/components/createContainer/createContainerController.js

```diff
@@ -232,7 +233,174 @@ function ($q, $scope, $state, $stateParams, $filter, Container, ContainerHelper,
     return config;
   }
 
+  function confirmCreateContainer(cb) {
+    Container.query({
+      all: 1,
+      filters: {name: [$scope.config.name]}
+    }, function (data) {
+      var confirmDialog = false;
+      var containerId;
+      // Prompt if we found name to confirm replacement
+      for (var c in data) {
+        for (var n in data[c].Names) {
+          if (data[c].Names[n] === '/' + $scope.config.name) {
+            confirmDialog = true;
+            containerId = data[c].Id;
+          }
+        }
+      }
+      if (confirmDialog) {
+        ModalService.confirmDeletion(
+          'A container with the same name is already present on this host. Do you want to remove it?',
+          function onConfirm(confirmed) {
+            if(!confirmed) { cb(false); }
+            else {
+              // Remove old container
+              Container.remove({id: containerId, v: 0, force: true}, function(d) {
```

Comment:
Use `ContainerService.remove()` to remove the container and the associated resource control.